### PR TITLE
Typings remove

### LINKED ISSUE #19: Update to typings instead of TypeScriptDefinitions
Similar to https://github.com/AnalyticalGraphicsInc/obj2gltf/pull/31 and https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/179

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes #19 

Removing typings/TypeScriptDefinitions entirely. We may go with a different solution in the future.

@shehzan10 can you review?

### CODE REVIEW

File: tools/gulpfile.js [hunk 1]
```diff
@@ -73,39 +73,14 @@ gulp.task('coverage', function () {
         ' cover' +
         ' --include-all-sources' +
         ' --dir coverage' +
-        ' -x "bin/** doc/** specs/** coverage/** index.js gulpfile.js"' +
+        ' -x "bin/** -x doc/** -x specs/** -x coverage/** -x index.js -x gulpfile.js"' +
```
Comment (shehzan10) [inline tools/gulpfile.js]: I think you should move this change into a separate commit.

**Comment by lilleyse** [comment]
@shehzan10 fixed.
